src/client/components/Sidebar/CryptoTrendingCharts.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Trend from 'react-trend';
+import fetch from 'isomorphic-fetch';
+import _ from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import Loading from '../Icon/Loading';
+import { getCryptoDetails } from '../../helpers/cryptosHelper';
+import USDDisplay from '../../components/Utils/USDDisplay';
+import './SteemTrendingCharts.less';
+
+const fetchCryptoPriceHistory = symbol =>
+  fetch(
+    `https://min-api.cryptocompare.com/data/histoday?fsym=${symbol}&tsym=USD&limit=7`,
+  ).then(res => res.json());
+
+class CryptoTrendingCharts extends React.Component {
```

Comment:
We should avoid having duplicate code, i think this component should also be used on the Steem and SBD market block on wallet pages.

Reply:
Steem & SBD market blocks are both combined into one, thats why I kept the SteemTrendingCharts component... 

I can make it look like this if you want? 

<img width="458" alt="screen shot 2017-12-12 at 4 18 10 pm" src="https://user-images.githubusercontent.com/5067716/33908974-0f9986f0-df58-11e7-84a6-99f25616997d.png">

and it's not really duplicate code, since if you look at steem trending charts component, there is a few different logic in there.

Reply:
Can't you just make that the component can take an array of symbol instead of only one?

Reply:
yeah I can but this would only be for one case? just for this specific scenario? Is that worth doing, when we are going to update the charts design anyways?

Reply:
If we are not doing this we are going to update the charts in 2 different places isn't it?

Reply:
yeah thats true, I'll see what I can come up with .